sdk/dotnet/Pulumi.Automation/Runtime/LanguageRuntimeService.cs

```diff
@@ -60,7 +61,14 @@ public override async Task<RunResponse> Run(RunRequest request, ServerCallContex
                 runner => this._callerContext.Program.InvokeAsync(runner, cts.Token))
                 .ConfigureAwait(false);
 
-            return new RunResponse();
```

Comment:
From Go I got used to early return being for the "exceptional" case not the happy case, don't know if this also makes sense in C# stylistically.

Reply:
I've seen it both ways, probably just a style decision. Up to you guys. I did it this way because Pulumi hasn't yet bumped their c# version such that we can use `is not null` and I find `is null` easier to read than `!= null`.

Reply:
Let's leave it as is then! ty.